### ISSUE TITLE
overflowing headline on mobile layout

### ISSUE THREAD

**Issue by thundernixon** [roberto-arista/PythonForDesigners#5]
Awesome work on this! It's beautiful, fun, and I bet a lot of designers will find this site helpful.

One thing I've noticed ...

When I open the site on my phone, coming from Twitter, the headline overflows the screen, introducing an unintentional horizontal scroll.

![image](https://user-images.githubusercontent.com/7355414/52295961-231e8700-294b-11e9-9223-22eac0713a25.png)

This is because the following CSS makes that headline `460px`, regardless of how narrow its container gets:

```
.opening h1 {
    width: 460px;
}
```

If I comment that out, it does look a bit awkward, but it's much better:

![image](https://user-images.githubusercontent.com/7355414/52296145-a0e29280-294b-11e9-9072-294fa923dff2.png)

If you want to make it significantly better, you could modify the font-size and margin-top:

![image](https://user-images.githubusercontent.com/7355414/52296299-fae35800-294b-11e9-9f46-2dc1ec8f33e5.png)

```
.opening h1 {
    font-size: 2em;
    margin-top: 2.25em;
}
```

You may want to play with some media queries to control this nicely across different headlines and devices. If you really want to get fancy, you could even try something like [CSS locks](https://fvsch.com/css-locks/). 

---

As another mobile layout thought, it might be nice to link to the GitHub repo on the mobile layout, rather than just having it in the "About" section. I wanted to make this issue sooner, but didn't initially realize that there was a public repo, as I didn't see it in the footer. 

---

Awesome work on this, though! Having done a bit of technical writing about getting started in Drawbot, I know it takes a *ton* of work to do well.

**Comment by roberto-arista** [comment]
I swear it was fine before the release 😅

**Comment by roberto-arista** [comment]
Thanks for pointing that out, it's a bit I don't browse the website with a mobile phone, I'll fix it today.

**Comment by roberto-arista** [comment]
I quickly scrolled through the CSS locks page, looks very interesting. I'll dive into it during the weekend :)

**Comment by thundernixon** [comment]
Haha, I know the feeling! No worries; most people using this seriously will want to do so at a computer. But yeah, making the headlines fluid width will solve most of the issue

**Comment by roberto-arista** [comment]
solved with fae90d339bd6fcfd5a2675e6caddc22d23516e6e

**Comment by roberto-arista** [comment]
thanks!
